code/modules/client/preferences.dm

```diff
@@ -569,7 +569,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				var/available_in_days = job.available_in_days(user.client)
 				HTML += "<font color=red>[rank]</font></td><td><font color=red> \[IN [(available_in_days)] DAYS\]</font></td></tr>"
 				continue
-			if((job_civilian_low & ASSISTANT) && (rank != "Assistant") && !jobban_isbanned(user, "Assistant"))
+			if((job_civilian_low) && (rank != SSjob.overflow_role) && !jobban_isbanned(user, SSjob.overflow_role))

```

Comment:
Need to figure out a way to properly check the overflow role flag for the first part of the if

I could set it manually as its own var but that would probably result in admins setting one var and not the other and breaking job setup

Reply:
simple semihacky solution: Create a helper proc that does both at once for admins.